### ISSUE TITLE
ucscTables() does not work properly for hs1 genome

### ISSUE THREAD

**Issue by hpages** [lawremi/rtracklayer#98]
Based on the online Table Browser at https://genome.ucsc.edu/cgi-bin/hgTables, the tables for track `NCBI RefSeq` for hs1 are: `hub_3671779_ncbiRefSeq`, `hub_3671779_ncbiRefSeqCurated`, `hub_3671779_ncbiRefSeqPredicted`, `hub_3671779_ncbiRefSeqOther`, and `hub_3671779_ncbiRefSeqPsl`.

However, with the latest **rtracklayer** (1.61.2), `ucscTables()` gives me:
```
library(rtracklayer)

ucscTables("hs1", "NCBI RefSeq")
# [1] "hub_3671779_hgUniqueHg38" "hub_3671779_hgUniquehg19"
```
Somethin' ain't right.

Thanks,
H.

**Comment by MurliNair** [comment]
I am also having issues accessing it. 
library(rtracklayer)
library(GenomicFeatures)
library(TxDb.Mmusculus.UCSC.mm10.knownGene)
mm10.refseq.db <- makeTxDbFromUCSC(genome="mm10", table="refGene")

Error in .local(.Object, ...) : Failed to obtain 'hguid' cookie

supportedUCSCtables(genome="hg19", url="http://genome.ucsc.edu/cgi-bin/")

Error in .local(.Object, ...) : Failed to obtain 'hguid' cookie

**Comment by hpages** [comment]
@MurliNair See issue #95 This has nothing to do with the `ucscTables()` issue for hs1.

**Comment by sanchit-saini** [comment]
For now, I have created a PR to fix it, though the root cause of this issue was kind of weird. ucscTableTracks() was making a request to the "https://genome-euro.ucsc.edu/" server, which was returning different results than "https://genome.ucsc.edu"

I thought the UCSC mirror pointed to the same genomic data, or am I missing something @hpages?

**Comment by hpages** [comment]
Thanks @sanchit-saini for looking into this.

> I thought the UCSC mirror pointed to the same genomic data,  or am I missing something @hpages?

I thought that too. Maybe @maximilianh can clarify?

Thanks

**Comment by maximilianh** [comment]
Yes. The three mirrors (ÇA, Bielefeld=Europe, Tokyo=Asia) are (should be)
identical. I hope to have a China mirror soon.

They only differ in a few tracks for a few minutes per day or week, when
OMIM etc are updated.

**Comment by jennyujin22** [comment]
Hello, I got an issue about Ucsc track, which is related to [#99 ](https://github.com/lawremi/rtracklayer/pull/99)

When I learn the code

`rTrack <- Gviz::UcscTrack(genome="hg19", chromosome=chrom, track = "NCBI RefSeq",
                    table = "ncbiRefSeq",
                    from=minbase, to=maxbase, trackType="GeneRegionTrack",
                    rstarts="exonStarts", rends="exonEnds", gene="name", 
                    symbol="name2", transcript="name", strand="strand", 
                    fill="darkblue",stacking="squish", name="RefSeq",
                    showId=TRUE, geneSymbol=TRUE)`

I got an error 

"Error in normArgTrack(track, trackids) : Unknown track: refSeqComposite"

When I did trace back, I thought that the error was more likely related to rtracklayer's ucsc.R (I found the function normArgTrack in this script) than Gviz, which supplied the UcscTrack I used.

Could anyone give some information about the error?

Many thanks,

Sincerely,

Yujin Kim.
